relation widget information (#31)
opengisch/QField-docs: en/project-management/vector-layers.rst

+-------------------+-----------------+-------------------------------------------------+
| Relation Reference| :yay:`✔`        |                                                 |
+-------------------+-----------------+-------------------------------------------------+
| Relation Widget   | :yay:`✔`        |                                                 |
+-------------------+-----------------+-------------------------------------------------+
| Others            | :nay:`✘`        | :ref:`make_it_grow`                             |
+-------------------+-----------------+-------------------------------------------------+

Relation Widget
................

> For more information checkt the description of relations at the official `QGIS Documentation <https://docs.qgis.org/2.18/en/docs/user_manual/working_with_vector/attribute_table.html#creating-one-or-many-to-many-relations>`_.

Reviewer: ```suggestion
For more information refer to the official `QGIS Documentation <https://docs.qgis.org/3.4/en/docs/user_manual/working_with_vector/attribute_table.html#creating-one-or-many-to-many-relations>`_.
```

.. container:: clearer text-center

  .. figure:: /images/relation_editor_widget_list.png
     :width: 500px

> The relation widget shows all the referencing child features in a list. On many-to-many relationships, according to the cardinality setting in the QGIS Vector Layer Properties, on clicking/deleting an entry in the list effects directly the child feature (on direct cardinality) or the linking table entry (on many-to-one cardinality). The second case is usually used when there are additional relation information (e.g. percentage) in the linking table. 

Reviewer: I would only put the first sentence here and the rest in a dedicated subsection "deleting children" (or "cascaded delete") at the end of the chapter.
I.e.

1. First a very short introduction to the topic.
2. Then some information about visualization (display expression etc)
3. Then the foreign key info
4. Cascaded delete
